Justfile

```diff
@@ -90,7 +90,11 @@ insert-version:
 	rm -rf {{dist_dir}}/web_modules
 	cp -rf web_modules {{dist_dir}}/web_modules
 	cp {{src_dir}}/Javascript/Main.js {{dist_dir}}/index.js
-	{{node_bin}}/esbuild --bundle --outfile={{dist_dir}}/worker.min.js {{src_dir}}/Javascript/Worker.js
+	{{node_bin}}/esbuild \
+		--bundle \
+		--define:API_ENDPOINT="$(jq .API_ENDPOINT config/{{config}}.json)" \
```

Comment:
Here's where the magic happens:
`jq` will pull a json value specified by the path out of a json file. `.` refers to the json's root, and then we look for the `API_ENDPOINT` key.

Then esbuild will replace `API_ENDPOINT` in the js source with the value returned from `jq` (note that jq will return the value quoted). [Also, here's a link to esbuild's define docs](https://esbuild.github.io/api/#define)